firmware_vex/nucleo/Makefile

```diff
@@ -1,10 +1,10 @@
 
-export PATH=$PATH:/bin:/usr/local/bin/:../../venv/bin:/usr/bin/:.
+#export PATH=$PATH:/bin:/usr/local/bin/:../../venv/bin:/usr/bin/:.
 
 #DEV=/dev/cu.usbmodem3103
-DEV=/dev/cu.usbmodem34203
+#DEV=/dev/cu.usbmodem34203
 #DEV=/dev/cu.usbmodem34103
-#DEV=/dev/ttyACM0
+DEV=/dev/ttyACM0
```

Comment:
maybe add a comment that this should be changed after identifying the device (also see #18)